scripts/network-launch/launch.sh

```diff
@@ -102,6 +109,24 @@ start_node () {
 EOT
 }
 
+clone_scripts () {
+  ip=$1
+  if [ $mode == "normal" ]; then
+  ssh ${user}@${ip} 'bash -s'  << EOT
+    sudo su -c "cd ${path}; git clone ${repo} ${dirName}; cd ${dirName}; git checkout ${repoBranch}; chmod -R 777 chain-data"  
+EOT
+  fi
+}
+
+stop_node_partially () {
```

Comment:
Why partially?

Reply:
We can rename another method. It says stop_node but it also deletes the folder

Reply:
Shall we do stop_node and stop_and_delete_node instead?

Reply:
Done